ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Node/GSM.hs

```diff
@@ -78,7 +77,7 @@ data CandidateVersusSelection =
     -- ^ Whether the candidate is better than the selection
   deriving (Eq, Show)
 
-data GsmView m upstreamPeer selection candidate = GsmView {
+data GsmView m upstreamPeer selection state = GsmView {
```

Comment:
`state` is maybe a bit too generic, what about `candidateState`/`chainSyncState`?

Reply:
29c5aaa3860c4c06a796169caf4a3e72d76f541d